styles/common/vazir.scss

```diff
@@ -0,0 +1,72 @@
+@font-face {
+  unicode-range: U+0600-06ff;

```

Comment:
@indeyets @indeyets It just load for persian character, and if the character is not in this range: U+0600-06ff, automatically it use the next font.